week-2/5/ex-5.html

```diff
@@ -0,0 +1,23 @@
+<div id="root"></div>
+<script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+<script crossorigin src="https://unpkg.com/prop-types@15.7.2/prop-types.js"></script>
+<script crossorigin src="https://unpkg.com/babel-standalone@6.26.0/babel.js"></script>
+
+<script type="text/babel">
+  function SayHello(props) {
+    return (
+      <div>
+        Hello {props.firstName} {props.lastName}!
+      </div>
+    );
+
+    something bad here
```

Comment:
You forgot HTML tags

Reply:
Hey @programmist i fixed that thing, is it better?